src/app/auth/styles.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+
+export const FormDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 2.5rem; // 40 px
```

Comment:
nit: can probably remove the px value comments and put a conversion factor that you used

Reply:
gotcha, i might  keep it as is for this PR, but i'll look out for this in future PRs!